github4s/shared/src/test/scala/github4s/integration/GHIssuesSpec.scala

```diff
@@ -93,6 +93,17 @@ trait GHIssuesSpec[T] extends BaseIntegrationSpec[T] {
     })
   }
 
+  "Issues >> RemoveLabel" should "return a list of labels" in {
```

Comment:
Should remove the label from the issue successfully?

Reply:
I went with `return a list of removed labels` since it actually returns the label that was removed in a list, wdyt?